hermes-tracker-elasticsearch/src/main/java/pl/allegro/tech/hermes/tracker/elasticsearch/consumers/ElasticsearchLogRepository.java

```diff
@@ -77,4 +80,63 @@ protected XContentBuilder notEndedDocument(MessageMetadata message, long timesta
                 .field(CLUSTER, clusterName);
     }
 
+    public static class Builder {
```

Comment:
yeah I know that is little a bit duplication but i think that is better than extending Builders